packages/contracts-bedrock/contracts/test/CommonTest.t.sol

```diff
@@ -111,6 +112,8 @@ contract L2OutputOracle_Initializer is CommonTest {
     function setUp() public virtual {
         _setUp();
```

Comment:
While you're in here:
```suggestion
        super.setUp();
```

Reply:
Addressed in this PR https://github.com/ethereum-optimism/optimism/pull/4938